app/services/cluster_stats/update_epoch_duration_service.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module ClusterStats
+  class UpdateEpochDurationService
+
+    NUMBER_OF_EPOCHS = 6
+
+    def initialize(network)
+      @network = network
+    end
+
+    def call
+      cluster_stat.update_attribute(:epoch_duration, epoch_duration)
+
+      cluster_stat
+    end
+
+    private
+
+    attr_reader :network
+
+    def cluster_stat
+      ClusterStat.find_or_create_by(network: network)
+    end
+
+    def epochs
+      @epochs ||= EpochWallClock.by_network(network).first(NUMBER_OF_EPOCHS)
+    end
+
+    def epoch_duration
+      diff = epochs.first.created_at - epochs.last.created_at
+      diffs_between_epochs_amount = epochs.size - 1
```

Comment:
If you must create a separate variable, which isn't really needed in this case, at least call it appropriately, eg. diff_between_epochs_sum
And then give diff variable (line 31) more meaning name as well, eg last_epochs_duration